ops/manifests/collector.py

```diff
@@ -50,35 +60,29 @@ def list_versions(self, event) -> None:
         }
         event.set_results(result)
 
-    def list_resources(self, event, manifests: Optional[str], resources: Optional[str]):
-        """List available, extra, and missing resources for each manifest."""
-        self._list_resources(event, manifests, resources)
-
```

Comment:
This is the breaking change @bschimke95 has identified.  Its implementation now will return something it previously did not.  I believe the correct way to do this is by bumping this python library to a next major rev.  I'll add that to this PR.  

`list_resources(...)` previously returned Nothing
`list_resources(...)` now will return something

there's nothing that was ever counting on this return value being there -- so this change is not REALLY breaking.  I can land either way on it though